src/tasks/airdrop.ts

```diff
@@ -135,12 +144,10 @@ task("build_airdrop_init_tx", "Creates a multisend transaction to assign multipl
             console.log(`Generating Full Tree`)
             const fullTree = generateFullTree(merkleLeaves)
             console.log(`Generating Proofs`)
-            for (const vesting of vestings) {
-                writeJson(`output/proofs/${vesting.vesting.account}.json`, {
-                    vesting: vesting.vesting,
-                    proof: extractProof(vesting.vestingHash, fullTree)
-                })
-            }
+            const jsonWriter = await spawn(new Worker("./writeProofWorker"))
+            const chunks = splitToChunks(vestings, 10000)
```

Comment:
Should `10000` be a constant and/or environment variable? JS doesn't like more than 11 threads and assumes it's a memory leak unless the limit is manually increased